pkg/controller/workload/cache/service_cache.go

```diff
@@ -19,6 +19,7 @@ package cache
 import (
 	"sync"
 
+	"istio.io/istio/pkg/log"
```

Comment:
Why not use `kmesh.net/kmesh/pkg/logger`?

Reply:
Good catch, IDE automatically completes it.